modules/afmongodb/afmongodb.c

```diff
@@ -360,20 +360,6 @@ _vp_process_value(const gchar *name, const gchar *prefix, TypeHint type,
   return FALSE;
 }
 
-static void
-_worker_retry_over_message(LogThrDestDriver *s, LogMessage *msg)
-{
-  MongoDBDestDriver *self = (MongoDBDestDriver *)s;
-
-  msg_error(
-    "Multiple failures while inserting this record into the database, "
-    "message dropped",
-    evt_tag_str("driver", self->super.super.super.id),
-    evt_tag_int("number_of_retries", s->retries.max),
-    evt_tag_value_pairs("message", self->vp, msg, self->super.seq_num,
-                        LTZ_SEND, &self->template_options));
-}
-
```

Comment:
Keep this `retry_over` callback (someone wants to know about the dropped `record`).